src/xebia_email_signature/signature.py

```diff
@@ -194,10 +202,15 @@ def get_theme(data: dict) -> dict:
     {'default': 'black', 'full_name': 'black', 'unit': 'black', 'link': 'black', 'logo_url': 'https://cdn.xebia.com/assets/logos/academy.png', 'url': 'https://xebia.com/training'}
     >>> get_theme({"unit": "Xebia Academy", "dark_theme": "off"})
     {'default': '#222222', 'full_name': '#6C1D5F', 'unit': '#5A5A5A', 'link': '#06A99C', 'logo_url': 'https://cdn.xebia.com/assets/logos/academy.png', 'url': 'https://xebia.com/training'}
+    >>> get_theme({"unit": "Xebia|Xpirit", "dark_theme": "off"})
+    {'default': '#222222', 'full_name': '#f28019', 'unit': '#5A5A5A', 'link': '#06A99C', 'logo_url': 'https://xpirit.com/wp-content/uploads/2023/04/Xebia_Xpirit_logo_2023_DEF.png', 'url': 'https://www.xpirit.com/'}
+    """
 
+    if (data["office"].__contains__("Xpirit")):
```

Comment:
The original implementation takes the theme from the field 'unit'.  So I would fill in `Cloud`  or `Xebia Academy`.

Can we do the same here?

Reply:
Within Xebia|Xpirit we have multiple units (Managed Service, Consultancy, CNSD). And we have "others" people that are not in a unit. This unit also is added to the signature.